ui/component/viewers/videoViewer/internal/videojs.jsx

```diff
@@ -431,7 +455,17 @@ export default React.memo<Props>(function VideoJs(props: Props) {
 
     // Cleanup
     return () => {
-      window.removeEventListener('keydown', curried_function);
+      const volumePanel = playerRef.current
+        .getChild(VIDEOJS_CONTROL_BAR_CLASS)
+        .getChild(VIDEOJS_VOLUME_PANEL_CLASS)
+        .el();
+
+      window.removeEventListener('keydown', keyDownHandlerRef.current);
+      containerRef.current.removeEventListener('wheel', videoScrollHandlerRef.current);
+
+      if (volumePanel.current) {
```

Comment:
`volumePanel.current` doesn't make sense, because it's not a React `ref` but a DOM element.

I think this just masks the crash by never calling 'remove' code?